courses/free-courses-bn.md

```diff
@@ -1,43 +1,44 @@
 ### Index
 
-* [Android](#android)
-* [C](#c)
-* [C#](#csharp)
-* [C++](#cpp)
-* [Competitive Programming](#competitive-programming)
-* [Dart](#dart)
-* [Docker](#docker)
-* [Flutter](#flutter)
-* [Git](#git)
-* [Go](#go)
-* [HTML and CSS](#html-and-css)
-    * [Bootstrap](#bootstrap)
-    * [Tailwind](#tailwind)
-* [IDE and editors](#ide-and-editors)
-* [Java](#java)
-* [JavaScript](#javascript)
-    * [Angular](#angular)
-    * [jQuery](#jquery)
-    * [Next.js](#nextjs)
-    * [Node.js](#nodejs)
-    * [React](#react)
-    * [Svelte](#svelte)
-    * [Vue.js](#vuejs)
-* [Kotlin](#kotlin)
-* [Linux](#linux)
-* [MongoDB](#mongodb)
-* [MySQL](#mysql)
-* [PHP](#php)
-    * [Laravel](#laravel)
-* [Programming paradigms](#programming-paradigms)
-* [Python](#python)
-    * [Django](#django)
-    * [Flask](#flask)
-* [Scratch](#scratch)
-* [Shell scripting](#shell-scripting)
-* [Swift](#swift)
-* [TypeScript](#typescript)
-* [WordPress](#wordpress)
+- [Index](#index)
+- [Android](#android)
+- [C](#c)
+- [C#](#c-1)
+- [C++](#c-2)
+- [Competitive Programming](#competitive-programming)
+- [Dart](#dart)
+- [Docker](#docker)
+- [Flutter](#flutter)
+- [Git](#git)
+- [Go](#go)
+- [HTML and CSS](#html-and-css)
+  - [Bootstrap](#bootstrap)
+  - [Tailwind](#tailwind)
+- [IDE and editors](#ide-and-editors)
+- [Java](#java)
+- [JavaScript](#javascript)
+  - [Angular](#angular)
+  - [jQuery](#jquery)
+  - [Next.js](#nextjs)
+  - [Node.js](#nodejs)
+  - [React](#react)
+  - [Svelte](#svelte)
+  - [Vue.js](#vuejs)
+- [Kotlin](#kotlin)
+- [Linux](#linux)
+- [MongoDB](#mongodb)
+- [MySQL](#mysql)
+- [PHP](#php)
+  - [Laravel](#laravel)
+- [Programming paradigms](#programming-paradigms)
+- [Python](#python)
+  - [Django](#django)
+  - [Flask](#flask)
+- [Scratch](#scratch)
+- [Shell scripting](#shell-scripting)
+- [Swift](#swift)
+- [TypeScript](#typescript)
+- [WordPress](#wordpress)
```

Comment:
You need to start fresh and don't edit this. This is already formatted as it is intended to be. If you are going to add an index or subindex it needs to follow the already established format.

If you keep changing this then the pull request will be rejected.